packages/server/e2e/step-definitions/evaluation.spec.ts

```diff
@@ -1,18 +1,39 @@
 import { defineFeature, loadFeature } from 'jest-cucumber';
-import { JsonValue, JsonObject, EvaluationDetails, EvaluationContext, ResolutionDetails, StandardResolutionReasons } from '@openfeature/shared';
+import {
+  JsonValue,
+  JsonObject,
+  EvaluationDetails,
+  EvaluationContext,
+  ResolutionDetails,
+  StandardResolutionReasons,
+  ProviderEvents,
+} from '@openfeature/shared';
 import { OpenFeature } from './../../src';
 // load the feature file.
 const feature = loadFeature('packages/server/e2e/features/evaluation.feature');
 
 // get a client (flagd provider registered in setup)
 const client = OpenFeature.getClient();
 
-const givenAnOpenfeatureClientIsRegisteredWithCacheDisabled = (given: (stepMatcher: string, stepDefinitionCallback: () => void) => void) => {
+const givenAnOpenfeatureClientIsRegisteredWithCacheDisabled = (
+  given: (stepMatcher: string, stepDefinitionCallback: () => void) => void
+) => {
   // TODO: when the FlagdProvider is updated to support caching, we may need to disable it here for this test to work as expected.
   given('a provider is registered with cache disabled', () => undefined);
 };
 
 defineFeature(feature, (test) => {
+  beforeAll((done) => {
+    client.addHandler(ProviderEvents.Ready, async () => {
```

Comment:
```suggestion
    client.addHandler(ProviderEvents.Ready, () => {
```

Reply:
Same as [here](https://github.com/open-feature/js-sdk/pull/436/files#r1205813869). Let me know I'm I'm wrong for some reason.